developing_packages/editable_packages.rst

```diff
@@ -129,13 +131,18 @@ the following syntax:
        src/cmp_a/include
```

Comment:
Could you include here the example with the for loop? I think it is useful to have a summary of the most common operations here